src/ui/mapFactory.ts

```diff
@@ -0,0 +1,26 @@
+import maplibregl from 'maplibre-gl';
+
+import RadarMap from './map';
+import RadarMarker from './marker';
+
+import type { RadarMapOptions, RadarMarkerOptions } from '../types';
+
+class MapUI {
```

Comment:
Lets keep this file as `/src/ui/map` as the main entrypoint to the maps stuff.

This can import the other files, and their file names can match the class names
```js
import RadarMap from './RadarMap';
import RadarMarker from './RadarMarker';
```